data/TusharPaul01.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Tushar Paul",
+  "bio": "IoT & AI/ML Developer : **Tech Enthusiast**. Find fun in innovation & creativity."
```

Please add the missing comma `,` here! 

```suggestion
  "bio": "IoT & AI/ML Developer : **Tech Enthusiast**. Find fun in innovation & creativity.",
```